### ISSUE TITLE
您好，很荣幸能够读到您的文章，但是在用您提供的代码训练网络时，我们无法复现您文章中的结果，这是因为您的网络在保存模型之后又以模型为基础训练了多次吗？又或者是其他的原因？期待您的回复，对此抱以万分的感谢！以下是我们训练的记录

### ISSUE THREAD

**Issue by JunJiang-Xiang** [yfsong0709/RA-GCNv1#3]
The running config is presented as follows:
path: /media/IVS/ff4b00bc-2738-4f64-a174-1828b85d80a2/skletons
config_id: 3303
resume: False
evaluate: False
extract: False
visualization: False
gpus: [0, 1, 2, 3, 4, 5, 6, 7]
seed: 1
visualize_sample: 0
visualize_class: 0
visualize_stream: 0
visualize_frames: []
subset: cv
max_frame: 300
batch_size: 16
data_transform: True
occlusion_part: []
occlusion_time: 0
pretrained: False
model_stream: 3
gcn_kernel_size: [9, 3]
drop_prob: 0.5
max_epoch: 50
learning_rate: 0.1
adjust_lr: [10, 30]
************************************************

Starting preparing ...
GPU0 used: 167.0M
GPU1 used: 10.0M
GPU2 used: 10.0M
GPU3 used: 10.0M
GPU4 used: 10.0M
GPU5 used: 10.0M
GPU6 used: 10.0M
GPU7 used: 10.0M
Successful!

Starting training ...
Epoch: 1/50, Training accuracy: 18.39%, Training time: 1171.3025s

Epoch: 2/50, Training accuracy: 40.96%, Training time: 2302.7678s

Epoch: 3/50, Training accuracy: 48.68%, Training time: 3445.6108s

Epoch: 4/50, Training accuracy: 54.57%, Training time: 4597.3663s

Epoch: 5/50, Training accuracy: 58.37%, Training time: 5747.0058s

Epoch: 6/50, Training accuracy: 61.77%, Training time: 6899.6089s

Epoch: 7/50, Training accuracy: 64.04%, Training time: 8035.9808s

Epoch: 8/50, Training accuracy: 65.88%, Training time: 9175.8494s

Epoch: 9/50, Training accuracy: 67.41%, Training time: 10332.6557s

Epoch: 10/50, Training accuracy: 68.31%, Training time: 11486.8736s

Epoch: 11/50, Training accuracy: 77.78%, Training time: 12629.4134s

Epoch: 12/50, Training accuracy: 79.65%, Training time: 13782.8078s

Epoch: 13/50, Training accuracy: 80.45%, Training time: 14920.9662s

Epoch: 14/50, Training accuracy: 80.91%, Training time: 16060.5800s

Epoch: 15/50, Training accuracy: 81.60%, Training time: 17213.3235s

Epoch: 16/50, Training accuracy: 81.94%, Training time: 18362.4075s

Epoch: 17/50, Training accuracy: 82.60%, Training time: 19545.8527s

Epoch: 18/50, Training accuracy: 82.61%, Training time: 20700.8370s

Epoch: 19/50, Training accuracy: 83.18%, Training time: 21845.3449s

Epoch: 20/50, Training accuracy: 83.46%, Training time: 22994.9654s

Epoch: 21/50, Training accuracy: 83.72%, Training time: 24149.8057s

Epoch: 22/50, Training accuracy: 83.80%, Training time: 25303.5228s

Epoch: 23/50, Training accuracy: 84.42%, Training time: 26452.8830s

Epoch: 24/50, Training accuracy: 84.53%, Training time: 27593.3021s

Epoch: 25/50, Training accuracy: 84.67%, Training time: 28743.2228s

Epoch: 26/50, Training accuracy: 84.93%, Training time: 29907.3569s

Epoch: 27/50, Training accuracy: 85.22%, Training time: 31049.8000s

Epoch: 28/50, Training accuracy: 85.41%, Training time: 32193.3246s

Epoch: 29/50, Training accuracy: 85.50%, Training time: 33340.7550s

Epoch: 30/50, Training accuracy: 85.81%, Training time: 34479.4433s

Epoch: 31/50, Training accuracy: 89.95%, Training time: 35613.0684s

Epoch: 32/50, Training accuracy: 90.66%, Training time: 36773.2559s

Evaluating for epoch 32 ...
Epoch: 32/50, Evaluating accuracy: 83.64%, Evaluating time: 37078.8465s

Epoch: 33/50, Training accuracy: 91.20%, Training time: 38232.6681s

Epoch: 34/50, Training accuracy: 91.32%, Training time: 39390.6272s

Evaluating for epoch 34 ...
Epoch: 34/50, Evaluating accuracy: 84.58%, Evaluating time: 39689.1361s

Epoch: 35/50, Training accuracy: 91.62%, Training time: 40847.2831s

Epoch: 36/50, Training accuracy: 91.88%, Training time: 41996.5480s

Evaluating for epoch 36 ...
Epoch: 36/50, Evaluating accuracy: 84.51%, Evaluating time: 42294.8288s

Epoch: 37/50, Training accuracy: 91.82%, Training time: 43449.6039s

Epoch: 38/50, Training accuracy: 92.21%, Training time: 44601.2630s

Evaluating for epoch 38 ...
Epoch: 38/50, Evaluating accuracy: 84.88%, Evaluating time: 44902.9284s

Epoch: 39/50, Training accuracy: 92.30%, Training time: 46054.4653s

Epoch: 40/50, Training accuracy: 92.64%, Training time: 47194.4850s

Evaluating for epoch 40 ...
Epoch: 40/50, Evaluating accuracy: 85.14%, Evaluating time: 47489.7674s

Epoch: 41/50, Training accuracy: 92.49%, Training time: 48627.1946s

Epoch: 42/50, Training accuracy: 92.64%, Training time: 49762.7207s

Evaluating for epoch 42 ...
Epoch: 42/50, Evaluating accuracy: 85.75%, Evaluating time: 50054.5987s

Epoch: 43/50, Training accuracy: 92.80%, Training time: 51204.0417s

Epoch: 44/50, Training accuracy: 93.01%, Training time: 52355.3827s

Evaluating for epoch 44 ...
Epoch: 44/50, Evaluating accuracy: 85.46%, Evaluating time: 52650.8600s

Epoch: 45/50, Training accuracy: 93.14%, Training time: 53808.8168s

Epoch: 46/50, Training accuracy: 93.31%, Training time: 54973.8655s

Evaluating for epoch 46 ...
Epoch: 46/50, Evaluating accuracy: 84.17%, Evaluating time: 55274.6766s

Epoch: 47/50, Training accuracy: 93.43%, Training time: 56420.0494s

Epoch: 48/50, Training accuracy: 93.51%, Training time: 57571.7087s

Evaluating for epoch 48 ...
Epoch: 48/50, Evaluating accuracy: 86.19%, Evaluating time: 57872.6788s

Epoch: 49/50, Training accuracy: 93.53%, Training time: 59015.8476s

Epoch: 50/50, Training accuracy: 93.72%, Training time: 60182.7554s

Evaluating for epoch 50 ...
Epoch: 50/50, Evaluating accuracy: 84.48%, Evaluating time: 60484.6964s

Finish training!
Best accuracy: 86.19%, Total time: 60484.8752s

Process finished with exit code 0

**Comment by yfsong0709** [comment]
尝试一下把pretrained参数改成True再试一试。另外就是可以适当地增加batch size，8个GPU情况下batch size设为16就太低了，每个卡上只跑了两个sample，可能会影响到model里的BN层吧。建议增加到64再尝试一下。

**Comment by JunJiang-Xiang** [comment]
感谢您的回复，我们将继续调试我们的代码。
 							
 		 			  
 
------------------ Original ------------------
